app/src/main/java/com/beemdevelopment/aegis/services/LaunchScannerTileService.java

```diff
@@ -0,0 +1,61 @@
+package com.beemdevelopment.aegis.services;
+
+import android.Manifest;
+import android.content.Intent;
+import android.content.pm.PackageManager;
+import android.os.Build;
+import android.service.quicksettings.Tile;
+import android.service.quicksettings.TileService;
+
+import androidx.annotation.RequiresApi;
+
+import com.beemdevelopment.aegis.R;
+import com.beemdevelopment.aegis.ui.MainActivity;
+
+@RequiresApi(api = Build.VERSION_CODES.N)
+public class LaunchScannerTileService extends TileService {
+    private static final int CODE_PERM_CAMERA = 0;
```

Comment:
This appears to be unused. I'd also suggest removing all of the empty overrides that are unused.